package.json

```diff
@@ -10,7 +10,8 @@
     "test:ui": "vitest --ui",
     "test:coverage": "vitest run --coverage",
     "lint": "eslint --ignore-path .gitignore --fix src",
-    "prettier": "prettier src --write"
+    "prettier": "prettier src --write",
+    "deploy": "npm run build && cdk bootstrap && cdk deploy"
```

Comment:
You shouldn't run cdk bootstrap every time when you build or deploy your app. It must be run once during configure environment for deploy to AWS.